packages/fx-core/src/component/local/localCertificateManager.ts

```diff
@@ -381,14 +382,15 @@ export class LocalCertificateManager {
   }
 
   private async checkCertificateWindows(thumbprint: string): Promise<boolean> {
+    const quotedThumbprint: string = shellQuote.quote([thumbprint]);
     try {
       // try powershell first
-      const getCertCommand = `Get-ChildItem -Path Cert:\\CurrentUser\\Root | Where-Object { $_.Thumbprint -match '${thumbprint}' }`;
+      const getCertCommand = `Get-ChildItem -Path Cert:\\CurrentUser\\Root | Where-Object { $_.Thumbprint -match '${quotedThumbprint}' }`;
```

Comment:
There's already single-quote in the command. Will this cause syntax error?

Reply:
The thumbprint has a fixed format, why quote is needed?

Reply:
Technically there is no need. To quote it just to meet the CodeQL checking.

Reply:
So would this change break the command?
E.g., if the `thumbprint` is `ABC123EF`, would the command be `... -match ''ABC123EF''`? Then the whole functionality is broken.